src/main/webapp/controllers/tutorialController.js

```diff
@@ -7,419 +7,521 @@ angular
+                                        content: "Press '?' to show all shortcuts. The shortcuts make a quick workflow possible and reduces the amount of clicks. For example, you can select an annotation and press 'a', 's', 'd' or 'f' to change the span word-wise. Try it out!",
                                         placement: "left"
                                     });
-                                    tour.addStep({
+                                    annotTour.addStep({
                                         path: "/discanno/#/annotation",
                                         element: "#1-anno-tool",
                                         title: "Annotation Tool",
                                         content: "Have fun with DiscAnno! The tour ends here :-)"
                                     });
                                 }
-                                
-                            } else {
-                                tour.addStep({
-                                    path: "/discanno/#/projects",
-                                    element: "#1-projects-header",
-                                    title: "Projects",
-                                    content: "Ohhh you have no assigned projects :-(. Tell your project manager to assign you projects."
-                                });
+
                             }
-                        });
-                    }
-                    
-                    // Start the tour
-                    $scope.startTour = function () {
-                        // Initialize the tour
-                        $rootScope.tour = tour;
-                        tour.init();
-                        tour.restart();
-                    };
-                    
+                        }
+                    });
                 }
-                
+                // Start the tours
+                $scope.startTourUserMgmt = function () {
+                    // Initialize the tour
+                    $rootScope.tour = $scope.userMgmtTour;
+                    $rootScope.tour.init();
+                    $rootScope.tour.restart();
+                    $rootScope.tour.start();
+                };
+                $scope.startTourSchemes = function () {
+                    // Initialize the tour
+                    $rootScope.tour = $scope.schemeTour;
+                    $rootScope.tour.init();
+                    $rootScope.tour.restart();
+                    $rootScope.tour.start();
+                };
+                $scope.startTourProjectExpl = function () {
+                    // Initialize the tour
+                    $rootScope.tour = $scope.peTour;
+                    $rootScope.tour.init();
+                    $rootScope.tour.restart();
+                    $rootScope.tour.start();
+                };
+                $scope.startTourAnnotator = function () {
+                    // Initialize the tour
```

Comment:
We should stick to BE or AE. Therefore we should either use "initialize" or "initialise" everywhere.